pkg/kv/bulk/sst_batcher.go

```diff
@@ -876,3 +890,23 @@ func createSplitSSTable(
 	right = &sstSpan{start: first, end: last.Next(), sstBytes: sstFile.Data()}
 	return left, right, nil
 }
+
+func (b *SSTBatcher) reportFlushStats(size sz, reason int, lastFlush, afterFlush time.Time) {
```

Comment:
What do you think about reworking this to emit the `ingestionStats` @dt has already fleshed out in the past? It holds a lot of valuable information that we can surface (maybe after some pre-processing) that is specific to this SSTBatcher that can be emitted every time it flushes or at some pre-defined interval. This would then get aggregated and displayed by the restore data processor.

Reply:
done, used ingestion stats as the exported event instead of a new event.